source/support.html.erb

```diff
@@ -134,7 +138,24 @@ title: Support
 
       <p>
         Outside of working hours, please use the escalation contact
-        details you have been sent.
+        details you have been sent.</p>
+
+        <h2 class="heading-large">Keep up to date</h2>
+
+        <p>
+        Our <a href="https://status.cloud.service.gov.uk/">system status page</a>
+        shows if live applications and databases are available. You can use this
+        page to sign up to get alerts and incident updates.
+      </p>
+
+      <p>
+        Join our <a href="https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en#!forum/gov-uk-paas-announce">announcements mailing list</a>
```

Comment:
Potentially could say "Click here to join..." just to make it explicitly clear how they join the mailing list.